pkg/cedar/intake/README.md

```diff
@@ -3,6 +3,16 @@
 This folder contains code for submitting data to the CEDAR Intake API for eventual storage in Alfabet. 
 The Intake API is designed to take a JSON string in its `Body` field, as well as a schema identifier in its `Schema` field.
 
+## Swagger
```

Comment:
This needs to be updated a bit, still; I also have a script I need to polish that adds a `"x-nullable": true` field on certain datetime fields in the Swagger, which is important for the generated code.

Reply:
Docs, script, and pre-commit hook updated/added as of https://github.com/CMSgov/easi-app/pull/1495/commits/2b31da3f665b8263de66f64582ee08cd8920d1d4.